From 0f5f478379c7b93beae3d88020fa517a549968fd Mon Sep 17 00:00:00 2001
From: Yasin Kilicdere <tyk@google.com>
Date: Tue, 24 Sep 2024 17:53:02 +0000
Subject: [PATCH] Fix UserRepository to use the correct callback method.

Bug: 369294724
Bug: 331853529
Bug: 360838273

Test: atest UserRepositoryImplTest
Flag: EXEMPT bugfix
(cherry picked from commit 7cb99687fe1a22dc515c303aac7ed74792113315)
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:175dd00420c6727793b724ecbb9fe90e54f8b688)
Merged-In: I8b412400b89f5c62ccfc314d49dad68400cb3e5b
Change-Id: I8b412400b89f5c62ccfc314d49dad68400cb3e5b
---
 .../systemui/user/data/repository/UserRepositoryImplTest.kt    | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packages/SystemUI/tests/src/com/android/systemui/user/data/repository/UserRepositoryImplTest.kt b/packages/SystemUI/tests/src/com/android/systemui/user/data/repository/UserRepositoryImplTest.kt
index 1af0f79f2dde..c10c5b46c935 100644
--- a/packages/SystemUI/tests/src/com/android/systemui/user/data/repository/UserRepositoryImplTest.kt
+++ b/packages/SystemUI/tests/src/com/android/systemui/user/data/repository/UserRepositoryImplTest.kt
@@ -275,8 +275,7 @@ class UserRepositoryImplTest : SysuiTestCase() {
 
             tracker.onBeforeUserSwitching(userId = 0)
             tracker.onUserChanging(userId = 0)
-            assertThat(selectedUser!!.selectionStatus)
-                .isEqualTo(SelectionStatus.SELECTION_IN_PROGRESS)
+            assertThat(selectedUser!!.selectionStatus).isEqualTo(SelectionStatus.SELECTION_IN_PROGRESS)
 
             // WHEN a profile change occurs while a user is changing
             tracker.onProfileChanged()
-- 
2.34.1

